src/test/java/uk/ac/cam/cl/dtg/isaac/quiz/IsaacItemQuestionValidatorTest.java

```diff
@@ -97,6 +97,26 @@ public final void isaacItemQuestionValidator_CorrectItems_CorrectResponseShouldB
         assertTrue(response.isCorrect());
     }
 
+    /*
+       Test that exact correct answers are recognised even if the choice is subset match.
+    */
+    @Test
+    public final void isaacItemQuestionValidator_CorrectItemsSubsetMatchEnabled_CorrectResponseShouldBeReturned() {
+
+        // Enable subset matching on the second choice in the list (the correct one)
+        ((ItemChoice) someItemQuestion.getChoices().get(1)).setAllowSubsetMatch(true);
```

Comment:
This will affect other tests, not just this one, and JUnit makes no promises to the order in which they are run. You should keep the test side effect free, and either initialise a new question at the start that will be useful for subset tests, or make a new question inside this test as you did later on.

Reply:
Why does this effect all tests? The @Before on the setUp function makes it run before each test, so `someItemQuestion` is initialised again - I wasn't sure either but I've just checked with an @After what the values of `allowSubsetMatch` are on the choices in `someItemQuestion`, and it doesn't affect the other tests as far as I can see